src/Core/Style.js

```diff
@@ -7,6 +7,33 @@ const cacheStyle = new Cache();
 const inv255 = 1 / 255;
 const canvas = document.createElement('canvas');
 
+// add label halo management once
+const labelStyleSheet = document.createElement('style');
+labelStyleSheet.type = 'text/css';
+labelStyleSheet.innerHTML = `.itowns-stroke-single:before {
```

Comment:
Could you add inline file loading to handling this css configuration (with a adding `specific itowns_stroke_single_before.ccs` file)?
See `glsl/json` file loading and `babel-plugin-inline-import` (https://github.com/iTowns/itowns/blob/master/.babelrc#L5)

Reply:
Good idea, done